src/components/pages/collectRecordFormPages/CollectRecordFormPageAlternative/CollectRecordFormPageAlternative.js

```diff
@@ -293,20 +294,23 @@ const CollectRecordFormPageAlternative = ({
       if (!isObservationsTable1ReducerInitialized) {
         loadObservationsFromCollectRecordIntoTableState({
           collectRecordBeingEdited,
+          formik,
           getPersistedUnsavedObservationsTableData: getPersistedUnsavedObservationsTable1Data,
+          isNewRecord,
+          observationsPropertyName: getObservationsPropertyNames(collectRecordBeingEdited)[0],
```

Comment:
first table na,e

Reply:
ahh I see - would be good to define as a var ... `const bleachingTable1 = getObservationsPropertyNames(collectRecordBeingEdited)[0]`